src/base/style/app.styl

```diff
@@ -46,9 +46,6 @@ body.mobile .no-mobile-margin
   border 1px solid lightgrey
   border-radius 0
 
-.accent-font
-  font-family $accentFont
-
```

Comment:
Ah yeah, I had somehow made the assumption that quasar had defined an accent font and we were only just overriding it before. Thanks!